src/components/onboarding/Intro/index.tsx

```diff
@@ -1,27 +1,155 @@
-import TempImage from '@/public/assets/graphics/cat404.png';
-import Image from 'next/image';
+import Allocation from '@/public/assets/graphics/onboarding/ACM_Fall23Allocation-JustinLu.jpg';
+import HoldBoba from '@/public/assets/graphics/onboarding/ACM_Fall24BitByteInfo_1-JustinLu.jpg';
+import Murou from '@/public/assets/graphics/onboarding/ACM_Fall24BitByteInfo_2-JustinLu.jpg';
+import Bonfire from '@/public/assets/graphics/onboarding/ACM_Fall24Bonfire_1-JustinLu.jpg';
+import RaymondBack from '@/public/assets/graphics/onboarding/ACM_Fall24Bonfire_2-JustinLu.jpg';
+import TiltKickoff from '@/public/assets/graphics/onboarding/ACM_Fall24Kickoff_1-JustinLu.jpg';
+import FocusPerson from '@/public/assets/graphics/onboarding/ACM_Fall24Kickoff_2-JustinLu.jpg';
+import KickoffBig from '@/public/assets/graphics/onboarding/ACM_Fall24Kickoff_3-JustinLu.jpg';
```

Comment:
Let's try and make these names as distinctive/descriptive as possible. I think
1. BitByteAllocation
2. BitByteInfoSession
3. BitByteSpeedFriending
4. BonfireMarshmallows
5. BonfireBeach (also I think this is Matthew, not Raymond) 👻 
6. KickoffSideView
7. KickoffCloseUp
8. KickoffFrontView